### DISCUSSION TITLE
Lit Elements Assets

### DISCUSSION THREAD

**Discussion by bsastregx** [lit/lit#3446] [Q&A]
Hi, I am new at Lit.

Many times, an element needs assets, like an icon, an image, a font.

1. Where should this assets be placed? 
2. How do I add the assets on the component.
3. How do I copy this assets for production?

I searched for "assets" in Lit website, without much success:

![image](https://user-images.githubusercontent.com/54677414/201175483-fe8ae63b-5686-4984-885a-3158414c40a5.png)

Thanks in advance.

**Comment by PonomareVlad** [comment]
Hey, @bsastregx 👋🏻

I think there are lot answers for every taste 🙂

If you are looking for a "classic" way, I recommend watching https://github.com/pwa-builder/pwa-starter repo

**Reply by bsastregx** [reply]
Hi @PonomareVlad 👋🏻
Thanks for the quick answer.
